package.json

```diff
@@ -66,191 +66,191 @@
-    "jest-extended": "4.0.2",
-    "jest-junit": "16.0.0",
-    "json5": "2.2.3",
-    "kleur": "^4.1.0",
-    "ky": "0.19.1",
-    "ky-universal": "0.6.0",
-    "lerna": "4.0.0",
-    "lint-staged": "15.2.2",
-    "lodash": "4.17.20",
-    "lost": "8.3.1",
-    "minimatch": "3.0.4",
-    "moment": "^2.24.0",
+    "gzip-size": "catalog:",
+    "html-validator": "catalog:",
+    "http-proxy": "catalog:",
+    "husky": "catalog:",
+    "image-size": "catalog:",
+    "is-animated": "catalog:",
+    "isomorphic-unfetch": "catalog:",
+    "jest": "catalog:",
+    "jest-environment-jsdom": "catalog:",
+    "jest-extended": "catalog:",
+    "jest-junit": "catalog:",
+    "json5": "catalog:",
+    "kleur": "catalog:",
+    "ky": "catalog:",
+    "ky-universal": "catalog:",
+    "lerna": "catalog:",
+    "lint-staged": "catalog:",
+    "lodash": "catalog:",
+    "lost": "catalog:",
+    "minimatch": "catalog:",
+    "moment": "catalog:",
     "nanoid": "3.1.30",
-    "next": "workspace:*",
-    "node-fetch": "2.6.7",
-    "node-plop": "0.31.1",
+    "next": "catalog:",
+    "node-fetch": "catalog:",
+    "node-plop": "catalog:",
     "npm-run-all": "4.1.5",
-    "nprogress": "0.2.0",
-    "octokit": "3.1.0",
-    "outdent": "0.8.0",
-    "pixrem": "5.0.0",
-    "playwright": "1.41.2",
+    "nprogress": "catalog:",
+    "octokit": "catalog:",
+    "outdent": "catalog:",
+    "pixrem": "catalog:",
+    "playwright": "catalog:",
     "playwright-chromium": "1.41.2",
-    "postcss": "8.4.31",
-    "postcss-nested": "4.2.1",
-    "postcss-pseudoelements": "5.0.0",
-    "postcss-short-size": "4.0.0",
-    "postcss-trolling": "0.1.7",
+    "postcss": "catalog:",
+    "postcss-nested": "catalog:",
+    "postcss-pseudoelements": "catalog:",
+    "postcss-short-size": "catalog:",
+    "postcss-trolling": "catalog:",
     "prettier": "3.2.5",
-    "prettier-2": "npm:prettier@^2",
-    "pretty-bytes": "5.3.0",
+    "prettier-2": "catalog:",
+    "pretty-bytes": "catalog:",
     "pretty-ms": "7.0.0",
-    "random-seed": "0.3.0",
+    "random-seed": "catalog:",
     "react": "19.0.0-rc-187dd6a7-20240806",
-    "react-17": "npm:react@17.0.2",
-    "react-builtin": "npm:react@19.0.0-rc-187dd6a7-20240806",
-    "react-dom": "19.0.0-rc-187dd6a7-20240806",
-    "react-dom-17": "npm:react-dom@17.0.2",
-    "react-dom-builtin": "npm:react-dom@19.0.0-rc-187dd6a7-20240806",
-    "react-dom-experimental-builtin": "npm:react-dom@0.0.0-experimental-187dd6a7-20240806",
-    "react-experimental-builtin": "npm:react@0.0.0-experimental-187dd6a7-20240806",
-    "react-is-builtin": "npm:react-is@19.0.0-rc-187dd6a7-20240806",
-    "react-server-dom-turbopack": "19.0.0-rc-187dd6a7-20240806",
-    "react-server-dom-turbopack-experimental": "npm:react-server-dom-turbopack@0.0.0-experimental-187dd6a7-20240806",
-    "react-server-dom-webpack": "19.0.0-rc-187dd6a7-20240806",
-    "react-server-dom-webpack-experimental": "npm:react-server-dom-webpack@0.0.0-experimental-187dd6a7-20240806",
```

Comment:
These need to remain since they're handled by `scripts/sync-react.js`. External tools also rely on them to read which version of React is currently vendored in the App Router

Reply:
Thank you, reverted those packages.